src/transcoder/transcoderoptionsflac.cpp

```diff
@@ -37,9 +39,30 @@ void TranscoderOptionsFlac::Load() {
   ui_->quality->setValue(s.value("quality", 5).toInt());
 }
 
+void TranscoderOptionsFlac::ValueChanged(int value) {
+  if (!IsInStreamingSubset(value)) {
+    ui_->info->setStyleSheet("QLabel { color : red }");
```

Comment:
I think you need to unset this in the `else` block.

Reply:
Done.